MakeMsix.targets

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Define the property MakeMsixVerbosity=high for additional logging (default=normal) -->
+  <PropertyGroup>
+    <MakeMsixVerbosity Condition="'$(MakeMsixVerbosity)' == ''">normal</MakeMsixVerbosity>
+    <MakeMsixSigned Condition="'$(MakeMsixSigned)' == ''">true</MakeMsixSigned>
```

Comment:
Is this always true, or would we ever want false?  I'm not suggesting getting rid of it.  I'm more curious if (like the above) it can be linked to some external force, but default to true here.  For instance, is there some way to say "if the property on the pipeline page is defined, then let it override this.  Otherwise default to true"?

Reply:
It's an override - when you create an MSIX package you can optionally also sign it. The default is true for the common case, but MSIX supports unsigned packages. This knob leaves an easy option to support that.

I thought about splitting this into 2 .targets but separating out signtool seemed unnecessary complexity.